build/GitCommitInfo.targets

```diff
@@ -27,7 +27,7 @@
 
     <WriteLinesToFile File="$(GitCommitInfoProps)"
                       Lines="$(GitCommitInfoPropsContent)"
-                      Condition=" '$(ShouldOverwriteThePropsFile)' == 'true' "
+                      Condition=" '$(ShouldOverWriteThePropsFile)' == 'true' "
```

Comment:
[https://www.merriam-webster.com/dictionary/overwrite](https://www.merriam-webster.com/dictionary/overwrite)

Overwrite is one word.  So "Write" shouldn't be capitalized.